src/constraints/constraint_connection_flow_capacity_strict.jl

```diff
@@ -0,0 +1,125 @@
+#############################################################################
+# Copyright (C) 2017 - 2018  Spine Project
+#
+# This file is part of SpineOpt.
+#
+# SpineOpt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# SpineOpt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#############################################################################
+
+"""
+    add_constraint_connection_flow_capacity_strict!(m::Model)
+
+Limit the maximum in/out `connection_flow` of a `connection` for all `connection_flow_capacity_strict` indices.
+
+Check if `connection_conv_cap_to_flow` is defined. The `connection_capacity` parameter is used to constrain the
+"average power" (e.g. MWh/h) instead of "instantaneous power" (e.g. MW) of the `connection`.
+For most applications, there isn't any difference between the two. However, for situations where the same `connection`
+handles `connection_flows` to multiple `nodes` with different temporal resolutions, the constraint is only generated
+for the lowest resolution, and only the average of the higher resolution `connection_flow` is constrained.
+If instantaneous power needs to be constrained as well, defining the `connection_capacity` separately for each
+`connection_flow` can be used to achieve this.
+"""
```

Comment:
Are new functions docstrings up to date? Not just copy/pasted? (Just making sure...)

Reply:
oops... they may have been copy/pasted lol - will fix them